newton/tests/test_import_usd.py

```
@@ -2424,6 +2424,188 @@ def PhysicsRevoluteJoint "Joint2"
         self.assertTrue(np.any(jnt_actgravcomp))
         self.assertTrue(np.any(~jnt_actgravcomp))
 
+    @unittest.skipUnless(USD_AVAILABLE, "Requires usd-core")
+    def test_ref_attribute_parsing(self):
+        """Test that 'mjc:ref' attribute for revolute and prismatic joints is parsed."""
+        from pxr import Usd  # noqa: PLC0415
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)